main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"math/rand"
+	"time"
+
+	"./createwindow"
+	"./keybinding"
+
+	"github.com/jezek/xgb/xproto"
+)
+
+const (
+	chromeWindowNumber = 5
+)
+
+func main() {
+	rand.Seed(time.Now().Unix())
+
+	session, err := createwindow.Setup(chromeWindowNumber)
+	if err != nil {
+		errorHandler(err)
+		return
+	}
+
+	if err := keybinding.UpdateMaps(session.X); err != nil {
+		errorHandler(err)
+		return
+	}
+
+	eventLoop(session)
+}
+
+func eventLoop(session *createwindow.Session) {
+	for {
+		ev, err := session.X.WaitForEvent()
+		if err != nil {
+			errorHandler(err)
+			continue
+		}
+
+		if ev == nil {
+			fmt.Println("connection interrupted")
+			session.ForceQuit()
+			return
+		}
+
+		switch ev.(type) {
+
+		case xproto.KeyPressEvent:
+			errorHandler(err)
```

Comment:
I don't get why this is, but I'm assuming its waiting on some other code change?

Reply:
this PR is just responsible for opening the windows, the event loop will be changed later

Reply:
yeah, this i added to make sure it's working but it will be replaced by the proper event handlers in another package